tests/unit/azure/test_credentials.py

```diff
@@ -542,3 +544,68 @@ def test_create_access_connectors_for_storage_accounts(sp_migration):
     assert len(validation_results) == 1
     assert validation_results[0].name.startswith("ac")
     assert len(validation_results[0].failures) == 0
+
+
+@pytest.fixture
+def sp_migration_1(installation, credential_manager):
+    ws = create_autospec(WorkspaceClient)
+    ws.secrets.get_secret.return_value = GetSecretResponse(
+        value=base64.b64encode("hello world".encode("utf-8")).decode("utf-8")
+    )
+
+    mock_get_secret = Mock()
```

Comment:
Technically, you don't need this variable. We're also preventing untyped mocks, because it's more difficult to refactor